compiler/rustc_const_eval/src/interpret/validity.rs

```diff
@@ -939,6 +942,64 @@ impl<'mir, 'tcx: 'mir, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
         self.validate_operand_internal(op, path, Some(ref_tracking), Some(ctfe_mode))
     }
 
+    #[inline(always)]
+    pub fn const_validate_mplace(
```

Comment:
I was quite happy to have this very const-specific code *not* in the `interpret` file. Why is it being moved?

Reply:
You mean like `const_validate_operand` 😆 

I can move it into the const_eval module, all I wanted was to make it a method

Reply:
Well the names are rather misleading, this one here is doing the full handling of recursive validation and error reporting.

Reply:
I moved it back to `const_eval.rs`